api/server/gin_middlewares.go

```diff
@@ -46,7 +46,7 @@ func traceWrap(c *gin.Context) {
 	// If wireContext == nil, a root span will be created.
 	// TODO we should add more tags?
 	serverSpan := opentracing.StartSpan("serve_http", ext.RPCServerOption(wireContext), opentracing.Tag{Key: "path", Value: c.Request.URL.Path})
-	serverSpan.SetBaggageItem("fn_appname", c.Param(api.CApp))
+	serverSpan.SetBaggageItem("fn_app", c.Param(api.CApp))
```

Comment:
fn_app_id ?

Reply:
this just kind of screws up every stat (just not nice if nobody is prepared), i don't think it's necessary to change this since the app name will be on every request still

Reply:
agreed, will revert